website/docs/d/cloud_project_user.html.markdown

```diff
@@ -0,0 +1,50 @@
+---
+layout: "ovh"
+page_title: "OVH: ovh_cloud_project_user"
+sidebar_current: "docs-ovh-datasource-cloud-project-user"
+description: |-
+  Get the details of a public cloud project user.
+---
+
+# ovh_cloud_project_user
+
+Get the user details of a previously created public cloud project user.
+
+## Example Usage
+
+```hcl
+resource "ovh_cloud_project_user" "user" {
+ service_name = "XXX"
+ description  = "my user"
+ role_names   = [
+  "objectstore_operator"
+ ]
+}
+
+data "ovh_cloud_project_user" "my_user" {
+ service_name = ovh_cloud_project_user.user.service_name
+ user_id      = ovh_cloud_project_user.user.id
```

Comment:
for example we can use the datasource with the list of users here in order to have the user_id or simply:

user_id      = "111111"